Android.mk

```diff
@@ -0,0 +1,75 @@
+# Mesa 3-D graphics library
```

Comment:
John Stultz has been telling me about this other work! Sounds like some good progress on getting things moving.
I guess this is not mesa 3-d graphics library here though ;-)

Reply:
Yes. It was a copy-paste from mesa3d. I spent a day to make it buildable for libcamera and to get quick feedback. Changed.
Can I use Apache-2.0 license for Android.mk files?